dashboard/app/views/levels/edit.html.haml

```diff
@@ -25,8 +25,9 @@
 
   / Only enter name to create new records of certain level types.
   - unless @level.new_record? && @level.is_a?(Blockly)
-    = render partial: 'levels/editors/instructions_area', locals: {f: f} unless @level.is_a? EvaluationMulti
+    = render partial: 'levels/editors/instructions_area', locals: {f: f} unless (@level.is_a?(EvaluationMulti) || @level.is_a?(DSLDefined))
     = render partial: 'levels/editors/code_area', locals: {f: f} if @level.uses_droplet? || @level.is_a?(Blockly)
+    = render partial: 'levels/editors/dsl', locals: {f: f} if (@level.is_a?(DSLDefined) && !(@level.is_a?(EvaluationMulti)))
```

Comment:
we should eventually be able to get rid of this weird logic for evaluation multi

Reply:
Working on in #35086